lib/__tests__/test.js

```diff
@@ -2,7 +2,45 @@
 
 const fs = require('fs');
 const path = require('path');
-const defineTest = require('jscodeshift/src/testUtils').defineTest;
+const {walkDir} = require('../execution-helper');
+
+/**
+ * @param {string} dirname - the testing directory
+ * @param {string} transformName - the name of the transform
+ * @param {string} filePrefix - the test file prefix
+ */
+function defineTest(dirname, transformName, filePrefix) {
+  const testName = `rewrites correctly on ${filePrefix}`;
+  describe(transformName, () => {
+    it(testName, () => {
+      runTest(dirname, transformName, filePrefix);
+    });
+  });
+}
+
+/**
+ * @param {string} dirname - the testing directory
+ * @param {string} transformName - the name of the transform
+ * @param {string} filePrefix - the test file prefix
+ */
+function runTest(dirname, transformName, filePrefix) {
```

Comment:
Let's make this a bit less generic by removing transformName. Then we can just require() the rewriter at the top of the file. We don't need infrastructure to test multiple different transforms.